index.html

```diff
@@ -35,7 +36,7 @@
           <i id="main-image" class="default"></i>
         </div>
         <div class="col-md-7 col-xs-12">
-          <h1>Il mio diario dell´allattamento</h1>
```

Comment:
"´" and "`" are used in italian to stress a letter, in this case we need an apostrophe so d'allattamento